src/foam/core/PropertyInfo.java

```diff
@@ -20,6 +20,7 @@
 
   public boolean getTransient();
   public boolean getRequired();
+  public Class getPropertyClass();
```

Comment:
Sure, that sounds like it would be clearer. I've updated to `getValueClass()`. Thanks.